addons/web_editor/static/src/less/web_editor.ui.components.less

```diff
@@ -203,21 +210,30 @@
 
 // MODALS
 body .modal {
+
+    // ALL MODALS
+    .o-w-preserve-tabs();
+
+    .modal-footer {
+        .btn {
+            text-transform: uppercase;
+        }
+    }
```

Comment:
Mmh, those will style client modals. I think that this should be moved to website where we style the `.o_website_modal`